backend/schema/jsonschema_test.go

```diff
@@ -307,7 +332,8 @@ func TestInvalidEnumValidation(t *testing.T) {
     "any": [{"name": "Name"}, "string", 1, 1.23, true, "2018-11-13T20:20:39+00:00", ["one"], {"one": 2}, null],
     "keyValue": {"key": "string", "value": 1},
 	"stringEnumRef": "B",
-	"intEnumRef": 3
+	"intEnumRef": 3,
+        "sumTypeRef": 10
```

Comment:
Where's the `@_kind`?

Reply:
Yes I see it in the schema, but it's not used in the _value_.

Reply:
Two cases where I don't think this will approach will work:

1. How do we differentiate between a sum type with Float and Int without a kind?
2. How does `@_kind` work if the type being discriminated is a map that can contain `@_kind` as a key.?

Reply:
Ahhhhh ok this finally make sense to me now. I'll think about this a bit more and try out whatever I come up with with the rest of the types to make sure we can discriminate.